Tasks/Gradle/package.json

```diff
@@ -8,8 +8,9 @@
     "url": "https://github.com/Microsoft/vso-agent-tasks/issues"
   },
   "dependencies": {
-    "xml2js": "^0.4.16",
+    "glob": "^7.1.0",
```

Comment:
Remove the ^ - the carret means that on build node will install any 7.x version of glob. But we only have LCA approval for 7.1.0